content/documentation.rst

```diff
@@ -62,20 +62,71 @@ semicolons, rather than after the Nth column.
     long lines.
 
 
-Writing User's Guides
-=====================
+About the Documentation Sections
+================================
+
+**Introduction**
+
+High level introduction to the software.
+
+
+**Installation and Upgrade Instructions**
+
+System requirements, installation, installation validation/procedures, upgrade
+procedures, uninstallation instructions.
+
+
+**Getting Started with SFTPPlus**
```

Comment:
I think that this can be left out.

the guides from styleguide should be as generic as possible... so that if tomorrow we will decide to work on another project, we can still use them and the information should still be valid.

I am still not 100% with this getting started with sftpplus guide... so I think that we can left it out of the styleguide recomandation